modules/balana-core/src/main/java/org/wso2/balana/ctx/RequestCtxFactory.java

```diff
@@ -166,16 +167,13 @@ public static RequestCtxFactory getFactory() {
      */
     public Element getXacmlRequest(String request) throws ParsingException {
 
-        Document doc;
-        ByteArrayInputStream inputStream;
-
-        inputStream = new ByteArrayInputStream(request.getBytes());
-        DocumentBuilderFactory  builder = Balana.getInstance().getBuilder();
```

Comment:
Balana.getInstance().getBuilder() returns a secured document builder factory. But instantiating balana engine to get the secured document builder seemed like an overkill when we already have a util method to do so :wink: